Merge dev to main (#128)
home-assistant/analytics.home-assistant.io: README.md

The worker source are located in the worker directory.

### Schedule

> As a part of the worker, there is several scheduled tasks that iterates over the stored data and makes a generated object that the site uses, these tasks run at different times.

Reviewer: ```suggestion
There are several scheduled tasks that iterate over the stored data and generate the data that the site uses. These tasks run at different times.
```

Author: That should not be committed her (directly to dev)

Author: https://github.com/home-assistant/analytics.home-assistant.io/pull/129 @balloob
